src/main/java/uk/gov/hmcts/cmc/claimstore/services/document/DocumentManagementService.java

```diff
@@ -93,7 +103,7 @@ public ClaimDocument uploadDocument(String authorisation, PDF pdf) {
                 .createdBy(OCMC)
                 .build();
         } catch (Exception ex) {
-            throw new DocumentManagementException(String.format("Unable to upload document %s to document management",
+            throw new DocumentManagementException(String.format("Unable to upload document %s to document management.",
```

Comment:
would be better to pass `ex` directly as DocumentManagementException(ex). as @Recover logging confirms that it has finally failed after retry.

Reply:
I think its best to throw a specific custom checked exception coz the root exception could have been anything.